docs/components/modeler/bpmn/business-rule-tasks/business-rule-tasks.md

```diff
@@ -9,24 +9,74 @@ modeled in [Decision Model and Notation](https://www.omg.org/dmn/) (DMN).
 
 ![task](assets/business-rule-task.png)
 
-Business rule tasks behave exactly like [service tasks](/components/modeler/bpmn/service-tasks/service-tasks.md). Both
-task types are based on jobs and [job workers](/components/concepts/job-workers.md).
+:::note
```

Comment:
:wrench: We could use `:::info` instead of `::note` because it highlights the text visually better than the gray `note` box.